src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { TodoWithUser } from '../../types';
+import { TodoInfo } from '../TodoInfo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  data: TodoWithUser[]
+};
+
+export const TodoList: React.FC<Props> = ({ data }) => (
+  <ul className="todo-list">
+    {data.map(todo => {
+      return (
+        <>
+          <li className="list-item">
+            <TodoInfo key={todo.id} todo={todo} />
+            <UserInfo key={todo.id} user={todo.user} />
+          </li>
+        </>
+      );
+    })}
```

Comment:
This return is redundant here, add unique key for each li item
```suggestion
    {data.map(todo => (
      <li className="list-item" key={...}>
          <TodoInfo key={todo.id} todo={todo} />
          <UserInfo key={todo.id} user={todo.user} />
       </li>
    ))}
```